tensorflow/lite/kernels/kernel_util.h

```diff
@@ -84,6 +84,17 @@ inline void SetTensorToDynamic(TfLiteTensor* tensor) {
   }
 }
 
+inline TfLiteStatus SetTensorSizes(TfLiteContext* context, TfLiteTensor* tensor,
```

Comment:
Declare only, and move definition in .cc file please

Thanks

Reply:
@karimnosseir , thanks for the review, i have updated the code as per the suggestion, kindly check and approve.

Regards
Amit